src/app/profile/overview/spaces/spaces.component.html

```diff
@@ -8,7 +8,10 @@ <h2 class="card-pf-title">
     </div>
   </div>
   <div class="card-pf-body f8-card-body">
-    <div *ngIf="spaces.length === 0; then showBlankState else showSpaces"></div>
+    <fabric8-loading-widget message="Please wait while we load the spaces"
+                            *ngIf="loading === true"></fabric8-loading-widget>
```

Comment:
`loading === true` and `loading === false` can be simplified to `loading` and `!loading` respectively. Or, `*ngIf="loading; else #loaded"` or something similar.